src/led.c

```diff
@@ -152,8 +153,8 @@ static inline void bl_led__set(
 	 * 2. Making bl_led_init() cache the clear masks so they can be reused
 	 *    here.
 	 */
-	gpio_clear(led_port[port], bl_led__get_pin_mask(port, 0xffff));
-	gpio_set(led_port[port], bl_led__get_pin_mask(port, led_mask));
+	gpio_port_set_masked(led_port[port], bl_led__get_pin_mask(port, 0xffff), '0');
```

Comment:
Probably need to convert from what's in `led_port` to the runtime device with `device_get_binding` first like below.